dataactbroker/README.md

```diff
@@ -1262,6 +1264,9 @@ This endpoint lists submissions for all agencies for which the current user is a
         - `publishing`
     - `certifying_user` - a string containing the name of the last user to certify the submission
     - `certified_on` - a string containing the last time/date the submission was certified. (`YYYY-MM-DD HH:mm:ss`)
+    - `quarterly_submission` - whether the submission is quarterly
+    - `window_end` - a string containing the last date of the submission window
+    - `time_period` - a string representing the time frame for the submission
```

Comment:
Putting this here but it's for this whole entry. None of this matches our current README style. It was an old one so we should update while we're here.